.eleventy.js

```diff
@@ -0,0 +1,11 @@
+const CleanCSS = require('clean-css')
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addFilter('cssmin', function (code) {
+    return new CleanCSS({}).minify(code).styles
```

Comment:
What's this do?  Is it just a css minifier?

Do we need it?  I think the amount of css we have... It probably does not matter.  And I bet I'd rather have the milliseconds of built time back (and smaller dependency tree) unless this really pulls a lot of weight somehow.

Reply:
That's true, ill drop it then